robot_poly: [ 1.0, 0.0, 0.0, 0.55,
             -1.0, 0.0, 0.0, 0.55,
              0.0, 1.0, 0.0, 0.55,
              0.0,-1.0, 0.0, 0.55,
              0.0, 0.0, 1.0, 0.15,
              0.0, 0.0,-1.0, 0.20]